react-components/src/components/RuleBasedOutputs/hooks/useEditRuleInstance.tsx

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { useFdmSdk } from '../../RevealCanvas/SDKProvider';
+import { RULE_BASED_OUTPUTS_VIEW } from '../constants';
+import { type ExternalIdsResultList, type RuleOutputSet } from '../types';
+import { fdmViewsExist } from '../../../utilities/fdmViewsExist';
+
+export const useEditRuleInstance = (): ((
+  ruleOutputSet: RuleOutputSet
+) => Promise<ExternalIdsResultList<unknown>>) => {
+  const fdmSdk = useFdmSdk();
+
+  const editCurrentRule = async (
```

Comment:
Wrap in a `useCallback`?

Reply:
ops changed here: 

https://github.com/cognitedata/reveal/pull/4356/commits/33ea6c18e924918d2307c5b6fe62fcaf19febcf7